.github/comments-aggregator/action.yml

```
@@ -0,0 +1,19 @@
+name: 'Merge Comments'
+description: 'Merge bot comment into one comment to keep PR page clean.'
```

Nit: should this be plural?
```suggestion
description: 'Merge bot comments into one comment to keep PR page clean.'
```